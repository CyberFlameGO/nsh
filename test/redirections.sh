# cd $(mktemp -d)
cd /tmp
rm -f message.txt
echo message1 > message.txt
echo message2 >> message.txt
ls > /dev/null | echo hello > hello.txt | hexdump -C
echo redirection > stderr.txt 2>&1
cat hello.txt
cat message.txt
cat stderr.txt
