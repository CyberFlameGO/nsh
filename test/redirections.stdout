hello
message1
message2
redirection
